config/database.yml

```diff
@@ -0,0 +1,85 @@
+# PostgreSQL. Versions 9.1 and up are supported.
```

Comment:
You should not keep config files on repo, only sample files
Ask mentors about that